app/js/components/presentational/LoginForm.jsx

```diff
@@ -9,7 +9,7 @@
 import React from 'react';
```

Comment:
@Charpell Is this file `LoginForm.jsx` part of the tests

Reply:
@patlub Yes